pkg/console/operator/sync_v400.go

```diff
@@ -213,7 +213,14 @@ func SyncSecret(co *consoleOperator, recorder events.Recorder, operatorConfig *o
 // therefore no additional error handling is needed here.
 func SyncConfigMap(co *consoleOperator, recorder events.Recorder, operatorConfig *operatorv1.Console, consoleConfig *configv1.Console, rt *routev1.Route) (*corev1.ConfigMap, bool, error) {
 	logrus.Printf("validating console configmap...")
-	cm, cmChanged, cmErr := resourceapply.ApplyConfigMap(co.configMapClient, recorder, configmapsub.DefaultConfigMap(operatorConfig, consoleConfig, rt))
+	defaultConfigmap, didMerge, configMergeMessage, err := configmapsub.DefaultConfigMap(operatorConfig, consoleConfig, rt)
+	if didMerge {
```

Comment:
I don't love having the `co.UpgradableWithUnsupportedConfigOverrides` down in here, I'd rather be moving these further up the stack into `starter.go`, but I can live with them so long as they stay within `sync_v400.go` until we have a good tech debt time to properly extract this all out to one place.

Reply:
Gone in next push.